packages/api-tests/reporter.js

```diff
@@ -0,0 +1,62 @@
+var mocha = require('mocha');
+module.exports = MyReporter;
+const testContainer = process.env.MOCHA_CONTAINER;
+var fs = require('fs');
+var path = require('path');
+
+let jsonOutput = [];
+const pattern = /(#[\w.()]+[\s]?)+/;
+
+function MyReporter(runner) {
```

Comment:
Nitpick: maybe give a more descriptive name for what the reporter does and move the reporter into a directory, so the root of the package doesn't become cluttered.